src/features/messages/queries/encoding.ts

```diff
@@ -70,3 +73,13 @@ export function searchValueToPostgresBytea(input: string): string | undefined {
     return undefined;
   }
 }
+
+export function isPotentiallyTransactionHash(input: string): boolean {
+  if (!input) return false;
+  return (
+    isValidTransactionHashEvm(input) ||
+    isValidTransactionHashCosmos(input) ||
+    isValidTransactionHashSealevel(input) ||
+    isValidTransactionHashStarknet(input)
```

Comment:
could instead iterate through ProtocolType enum and call `isValidTransactionHash`? more future-vm compatible